Bottom: ccc5c0493ee56b604cdddff93381d25b6e4269d4
Top:    54e49c8975d43110a93b3b6350d720fc4561ce4e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 21:44:35 +0000

Refresh of free_atexit-c-add-leaky_calloc

---

diff --git a/src/free_atexit.c b/src/free_atexit.c
index 3795c83..501775c 100644
--- a/src/free_atexit.c
+++ b/src/free_atexit.c
@@ -22,8 +22,8 @@
  *
  */
 
-
 #include <stdlib.h>
+#include <string.h>
 
 /* we need one ID per context - short int with 64k IDs should be enough */
 typedef unsigned int leaky_idx_t;
